.travis.yml

```diff
@@ -30,7 +30,7 @@ install:
   - source activate $ENV_NAME
 
   # Download Iris 2.0 and all dependencies.
-  - conda install -c conda-forge iris=2.1 cftime=1.0.1 numpy=1.15.4
+  - conda install -c conda-forge iris=2.2 cftime=1.0.1 numpy=1.15.4
```

Comment:
Might also want to update comment as well (or remove it entirely, it's obvious).

Reply:
I've updated this comment.